src/sentry/search/eap/spans.py

```diff
@@ -310,6 +310,61 @@ def _resolve_having(
             return parsed_terms[0], resolved_contexts
         return None, []
 
+    def _handle_virtual_column_terms(
+        self,
+        term: event_search.SearchFilter,
+        resolved_column: ResolvedColumn,
+        context: VirtualColumnContext | None,
+    ) -> tuple[ResolvedColumn, float | datetime | Sequence[float] | Sequence[str]]:
+        raw_value = term.value.raw_value
+        if context is None or isinstance(raw_value, datetime):
```

Comment:
Why do you need `isinstance(raw_value, datetime)`? Is something special about `datetime` values and columns requiring virtual column contexts?